src/cljs/main/broadfcui/page/workspace/notebooks/notebooks.cljs

```diff
@@ -65,6 +65,11 @@
 (defn create-inline-form-label [text]
   [:span {:style {:marginBottom "0.16667em" :fontSize "88%"}} text])
 
+(defn- leo-notebook-url [cluster]
+  (str (config/leonardo-url-root) "/notebooks/" (:googleProject cluster) "/" (:clusterName cluster)))
+
+(defn- contains-statuses [clusters statuses]
+  (not-empty (clojure.set/intersection (set statuses) (set (map #(:status %) clusters)))))
```

Comment:
* Import `[clojure.set :as set]` and then `set/intersection`
* `#(:status %)` -> `:status`
* Use `seq` instead of `not-empty`